duckduckgo/content-scope-scripts
# Special error page - malware support - translations

**Asana Task/Github Issue:** https://app.asana.com/0/1206594217596623/1208958530747145/f

## Description

Translations for phishing and malware error page

## Testing Steps

1. Visit the [preview](https://deploy-preview-1337--content-scope-scripts.netlify.app/build/pages/special-error/?locale=pt&errorId=malware) (change the locale parameter to any supported language)
2. Confirm that the malware error page does not show any English strings 
<img width="548" alt="image" src="https://github.com/user-attachments/assets/7eb5a202-dd30-4708-9b94-360f85217090" />
 

## Checklist


*Please tick all that apply:*
